### ISSUE TITLE
Install Steps Repeated Twice

### ISSUE THREAD

**Issue by pattisdr** [ethyca/fidesops#1026]
### Bug Description

It took me about twice as long as usual to get my local integration environment up and running from scratch this morning.  (Actually it was about an hour and a half total because I spent a lot of time trying to get it to run without pruning my volumes, but at some point had to cut my losses.)

#### Steps to Reproduce

 I tried to run the integration environment with postgres and mongo from a clean slate:
```
docker system prune --all -f --volumes
nox -s dev -- postgres mongodb
This now seems to install the requirements twice.  It goes through 17/17 backend steps, then does the 1/1 dev step of pip install -e ., which seems to trigger running the 17/17 backend steps again.
```

The second run through is able to use some of the cached information but it does seem to run the `pip install -r requirements.txt` step twice and have to reinstall everything.  It took about twice as long as usual to run this.

### Expected behavior

_A description of what you expected to happen._

### Screenshots

_If applicable, add screenshots to help explain your problem._

### Environment

* Version:
* OS:
* Python Version:
* Docker Version:

### Additional context

_Add any other context about the problem here._

**Comment by ThomasLaPiana** [comment]
@pattisdr you're totally correct, the build is happening twice but this issue is fixed by #966 , which should get merged any day now

**Comment by pattisdr** [comment]
wonderful thanks @ThomasLaPiana
